app/src/main/java/com/iron/espresso/presentation/home/setting/SettingAdapter.kt

```diff
@@ -0,0 +1,40 @@
+package com.iron.espresso.presentation.home.setting
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.iron.espresso.R
+import kotlinx.android.synthetic.main.setting_category_layout.view.*
+
+class SettingAdapter (
+    private val context: Context,
+    private val item: List<CategoryItem>) :
+    RecyclerView.Adapter<SettingAdapter.ViewHolder>(){
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+        val view = LayoutInflater.from(context)
+            .inflate(R.layout.setting_category_layout,parent,false)
+
+        return ViewHolder(view)
+    }
+
+    override fun getItemCount(): Int {
+        return item.count()
+    }
+
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+        val itemList = item[position]
+        holder.categoryTitle.text = itemList.categoryTitle
+        holder.categoryRecyclerView.layoutManager = LinearLayoutManager(context)
+        holder.categoryRecyclerView.adapter = SettingItemAdapter(context,itemList.items)
+
+    }
+
+    class ViewHolder(private val view: View) : RecyclerView.ViewHolder(view) {
+        val categoryTitle: TextView = view.category_title
+        val categoryRecyclerView: RecyclerView = view.category_recyclerview
```

Comment:
뷰 홀더에서 findViewById 하지않고 kotlin-extensions 로 접근하면 성능이슈가 있어요. 또 생성자에서 view를 필드 선언하지 않아도, 뷰홀더 자체에 변수 `itemView`가 있어요.
두개 링크 읽어보세요
https://www.androidhuman.com/lecture/kotlin/2017/11/26/kotlin_android_extensions_on_viewholder/
https://medium.com/@junsanghan/android-ktx-viewholder-layoutcontainer-baf21b43cce8
```suggestion
    class ViewHolder(view: View) : RecyclerView.ViewHolder(view) {
        val categoryTitle: TextView = itemView.findViewById(R.id.category_title)
        val categoryRecyclerView: RecyclerView = itemView.findViewById(R.id.category_recyclerview)
```

Reply:
viewHolder 에서도 databinding 사용하는거 @wswon 님한테 알려달라고 해보세요 ㅋㅋㅋ